app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -35,6 +35,7 @@ def prepare_for_clone_task
       :datastore     => dest_datastore,
       :folder        => dest_folder,
       :pool          => dest_resource_pool,
+      :profile       => dest_storage_profile,
```

Comment:
Sounds good to me, I called it `:profile` because that is what it is called in `VirtualMachineRelocateSpec` but `:storage_profile` will be more obvious